packages/flutter/lib/src/widgets/container.dart

```diff
@@ -356,6 +357,9 @@ class Container extends StatelessWidget {
   /// The transformation matrix to apply before painting the container.
   final Matrix4? transform;
 
+  /// /// The transform alignment for the transfrom property.
```

Comment:
This doc comment needs more details describing what this alignment is and also that it is only used when [transform] is not null.